src/components/calculators/show/output.js

```diff
@@ -9,27 +10,69 @@ const PrecisionNumber = ({value, precision, number=numberShow(value, precision)}
   </span>
 )
 
-const RangeDisplay = ({range: [low, high]}) => (
+const RangeDisplay = ({low, high}) => (
   <div><PrecisionNumber value={low}/> to <PrecisionNumber value={high}/></div>
 )
 
-const ResultSection = ({length, mean, adjustedConfidenceInterval}) => (
-  length === 1 ? <PrecisionNumber value={mean} precision={6}/> : <RangeDisplay range={adjustedConfidenceInterval}/>
+const ResultSection = ({length, mean, percentiles}) => (
+  length === 1 ? <PrecisionNumber value={mean} precision={6}/> : <RangeDisplay low={percentiles[5]} high={percentiles[95]}/>
 )
 
-export const Output = ({metric: {name, simulation}}) => (
-  <div className='output'>
-    <div className='row'>
-      <div className='col-xs-12 col-sm-7'>
-        <div className='name'>
-          {name}
-        </div>
-      </div>
-      <div className='col-xs-12 col-sm-5'>
-        <div className={`result-section${!_.isEmpty(_.get(simulation, 'sample.errors')) ? ' has-errors' : ''}`}>
-          {_.has(simulation, 'stats') && <ResultSection {...simulation.stats} />}
+const AnalyticsSection = (stats) => {
+  return (
+    <div className='stats-summary'>
+      {`According to the model, this value has a 95% chance of being above `}
+      <PrecisionNumber value={stats.percentiles[5]}/>
+      {` and a 95% chance of being below `}
+      <PrecisionNumber value={stats.percentiles[95]}/>
+      {'.'}
+      {` The mean value is `}
+      <PrecisionNumber value={stats.mean}/>
+      {` and the median is `}
+      <PrecisionNumber value={stats.percentiles[50]}/>
+      {`.`}
+    </div>
+  )
+}
+
+export class Output extends Component {
+  state = {
+    showAnalysis: false
+  }
+
+  showAnalysis(show=true) {
```

Comment:
I would either call this `toggleAnalysis()` and have it just flip the bit, or have two one-liner functions for `showAnalysis()` and `hideAnalysis()`, but it's not that important. I only mention it b/c I had other comments anyways.